messages/en.json

```diff
@@ -2,106 +2,202 @@
+        "Before the rope is pulled, \u0002{0}\u0002 shouts angrily at the crowd.",
+        "The villagers drag \u0002{0}\u0002 to the edge of a cliff. Before they can protest, they slip and fall into the abyss.",
         "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered with the leftover remains of their late comrade.",
+        "The villagers angrily behead \u0002{0}\u0002.",
+        "\u0002{0}\u0002 is thrown to a pack of hungry animals. They doesn't last long.",
+        "The village decides to feed \u0002{0}\u0002 to a den of lions. Soon nothing but bones are left.",
         "For SCIENCE, the villagers throw \u0002{0}\u0002 into a volcano. They discover that \u0002{0}\u0002's melting point is less than that of lava.",
-        "The angry villagers grab \u0002{0}\u0002 by the legs and arms, and throw them into a large fire. It seems that this doesn't burn that well, after all."
+        "For SCIENCE, the villagers electrocute \u0002{0}\u0002. They discover that \u0002{0}\u0002's body fried quite well.",
+        "For SCIENCE, the villagers give \u0002{0}\u0002 an acid bath. They discover that \u0002{0}\u0002's body was too weak to avoid decomposition.",
+        "The villagers tie \u0002{0}\u0002 to a bunch of balloons. They watch as \u0002{0}\u0002 floats off into the sunset.",
+        "The village decides to exile \u0002{0}\u0002. After they pack up and leave, nobody ever sees them again.",
+        "The angry villagers grab \u0002{0}\u0002 by the legs and arms, and throw them into a large fire. It seems that this doesn't burn that well, after all.",
+        "The angry villagers tie \u0002{0}\u0002 to a pole and burn them at the stake. It seems that this doesn't burn that well, after all."
     ],
     "quit_reveal": [
         "\u0002{0}\u0002, a{1} \u0002{2}\u0002, suddenly falls over dead before the astonished villagers.",
+        "\u0002{0}\u0002, a{1} \u0002{2}\u0002, suddenly succumbs to a fatal heart attack.",
         "While wearing a fake pair of antlers, \u0002{0}\u0002, a{1} \u0002{2}\u0002, is shot dead by a hunter.",
         "Standing under a tree, \u0002{0}\u0002, a{1} \u0002{2}\u0002, is killed by a falling branch.",
         "\u0002{0}\u0002, a{1} \u0002{2}\u0002, is killed by lightning before the villagers' eyes. The air smells of burnt flesh.",
         "Rampaging through the village, a bull gores \u0002{0}\u0002, a{1} \u0002{2}\u0002.",
         "\u0002{0}\u0002, a{1} \u0002{2}\u0002, falls into a vat of molasses and drowns.",
         "A pack of wild animals sets upon \u0002{0}\u0002. Soon the \u0002{2}\u0002 is only a pile of bones and a lump in the beasts' stomachs.",
+        "A pack of wild animals sets upon \u0002{0}\u0002. It seems a{1} \u0002{2}\u0002 makes for a good meal.",
+        "\u0002{0}\u0002, a{1}, \u0002{2}\u0002, choked to death on a lifesaver gummy and has died.",
```

Comment:
"LifeSaver" is a brand and I'd rather not have that (message itself is fine though).